redwood-treehouse-lazylayout-uiview/src/commonMain/kotlin/app/cash/redwood/treehouse/lazylayout/uiview/UIViewLazyList.kt

```diff
@@ -18,92 +18,136 @@ package app.cash.redwood.treehouse.lazylayout.uiview
 import app.cash.redwood.LayoutModifier
 import app.cash.redwood.treehouse.AppService
 import app.cash.redwood.treehouse.TreehouseApp
-import app.cash.redwood.treehouse.TreehouseContentSource
-import app.cash.redwood.treehouse.TreehouseUIKitView
 import app.cash.redwood.treehouse.TreehouseView.WidgetSystem
-import app.cash.redwood.treehouse.ZiplineTreehouseUi
-import app.cash.redwood.treehouse.bindWhenReady
-import app.cash.redwood.treehouse.lazylayout.api.LazyListInterval
 import app.cash.redwood.treehouse.lazylayout.widget.LazyList
+import app.cash.redwood.widget.Widget
+import kotlinx.cinterop.ObjCClass
 import platform.Foundation.NSIndexPath
-import platform.QuartzCore.CALayer
+import platform.Foundation.classForCoder
 import platform.UIKit.UITableView
 import platform.UIKit.UITableViewCell
 import platform.UIKit.UITableViewCellStyle
-import platform.UIKit.UITableViewDiffableDataSource
+import platform.UIKit.UITableViewCellStyle.UITableViewCellStyleDefault
+import platform.UIKit.UITableViewDataSourceProtocol
+import platform.UIKit.UITableViewDelegateProtocol
 import platform.UIKit.UIView
-import platform.UIKit.row
-import platform.UIKit.section
+import platform.UIKit.item
 import platform.darwin.NSInteger
+import platform.darwin.NSObject
 
 internal class UIViewLazyList<A : AppService>(
   treehouseApp: TreehouseApp<A>,
   widgetSystem: WidgetSystem,
 ) : LazyList<UIView> {
-  private val dataSource = TableViewDataSource(treehouseApp, widgetSystem)
-  private val root = UITableView().apply {
-    this.dataSource = this@UIViewLazyList.dataSource
+  private val reuseIdentifier = "cell"
```

Comment:
```suggestion
  private val reuseIdentifier get() = "cell"
```
Or `const` at top-level in file?